### DISCUSSION TITLE
Any SVG to d2 diagram

### DISCUSSION THREAD

**Discussion by edgett** [terrastruct/d2#401] [Feature Requests]
It would be super cool, if I could convert any SVG, I'm thinking from lucidchart, to d2 syntax.

This is not related to: https://github.com/terrastruct/d2/discussions/163

I'd imagine one would use a neural net to accomplish the transformation from svg to syntax.

**Comment by soma-m** [comment]
That sounds like a lot of work.

**Comment by alixander** [comment]
yeah i'm closing this, outside the scope of D2
